src/qililab/instruments/quantum_machines/quantum_machines_cluster.py

```diff
@@ -387,6 +391,11 @@ def get_parameter_of_bus(self, bus: str, parameter: Parameter):
             return self._qm._elements[bus].input.gain  # type: ignore[union-attr] # pylint: disable=protected-access
         if parameter == Parameter.IF:
             return self._qm._elements[bus].intermediate_frequency  # pylint: disable=protected-access
+        if parameter == Parameter.THRESHOLD_ROTATION:
```

Comment:
Note that set and get parameters will be changed to be able to do them while disconnected so you also have to take into account that this set and get THRESHOLD_ROTATION should work always.

Based on https://github.com/qilimanjaro-tech/qililab/pull/751

Reply:
Yes the getter and setter for that parameter does not need the instrument to be connected at all

Reply:
Also thanks for pointing out that PR I did not notice we made the same change. I will wait for it to be merged before then